test-app/app/src/main/assets/app/tests/testURLSearchParamsImpl.js

```diff
@@ -0,0 +1,56 @@
+describe("Test URLSearchParamsImpl ", function () {
+
+
+it("Test URLSearchParamsImpl keys", function(){
+const params = new URLSearchParamsImpl("foo=1&bar=2");
```

Comment:
As this line appears identical in each test, there may be some merit in hoisting the `params` variable up to the `describe` block, or alternatively initialising it in a `before` or `beforeEach` hook, but it's not a blocker.